### PR TITLE
Edit profile interface

### LINKED ISSUE #489: If you run update_profile or update_database with no arguments, you get a SQL syntax error
Be more defensive / provide a more helpful error in response.

### PR DESCRIPTION
**Pull request by marcua**
Part of #491

Also resolves #489 by testing for empty update_database or update_profile calls

### CODE REVIEW

File: tests/utils/ayb.rs [hunk 1]
```diff
@@ -239,14 +239,24 @@ pub fn update_database(
...
+    let mut cmd = Command::cargo_bin("ayb")?;
+    cmd.args(["client", "--config", config, "update_database", database])
+        .env("AYB_API_TOKEN", api_key);
 
-    cmd.stdout(format!("{result}\n"));
+    if let Some(level) = public_sharing_level {
+        cmd.arg("--public_sharing_level").arg(level);
+    }
+
+    if public_sharing_level.is_some() {
```
Comment (marcua) [inline tests/utils/ayb.rs:253]: combine with the if above.
